src/btree/bt_io.c

```diff
@@ -303,20 +303,7 @@ __wt_bt_write(WT_SESSION_IMPL *session, WT_ITEM *buf, uint8_t *addr, size_t *add
     if (encrypted)
         F_SET(dsk, WT_PAGE_ENCRYPTED);
 
-    /*
```

Comment:
Can we really remove this? From the comment, it seems it is related to salvage.

Reply:
I moved this comment into the place where the page header is updated with the write generation number.